### DISCUSSION TITLE
about_me/%E5%85%B3%E4%BA%8E%E5%8D%9A%E5%AE%A2

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#489] [Announcements]
# about_me/%E5%85%B3%E4%BA%8E%E5%8D%9A%E5%AE%A2

相关信息 就是玩儿 写在前面 本博客依赖vuepress-theme-hope主题模板进行搭建，依照官方教程做了些自定义的配置，大致符合自己简洁、清晰的审美理念，就直接上线了。此文档更多的是整体搭建和详细部署的说明，对自定义配置各位参照文档，一两个小时也就搞定了~ 助力每一个想要搭建自己博客的少男少女，愿你少踩点非必要的坑~ 博客相关内容持续集成中，欢迎关注交流！！

https://ru12321.github.io/about_me/%E5%85%B3%E4%BA%8E%E5%8D%9A%E5%AE%A2.html

**Comment by ru12321** [comment]
评论交流？
